indexing/pom.xml

```diff
@@ -134,5 +134,10 @@
       <artifactId>JUnitParams</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.api-client</groupId>
+      <artifactId>google-api-client</artifactId>
+      <version>2.2.0</version>
```

Comment:
how did we decide on the version? Is this the latest version?

Reply:
Yes this is the latest version of google-api-client.